packages/slice-machine/lib/models/common/widgets/Group/ListItem/index.js

```diff
@@ -59,10 +59,15 @@ const CustomListItem = ({
     setSelectMode(false);
   };
 
+  /* eslint-disable */
```

Comment:
ignoring the code block ?

We can ignore specific errors if you want but the whole file is too much, we wouldn't know if we introduce new errors

Reply:
Especially if you introduced new errors, you wouldn't know.

Reply:
yeah but since the linter in combined with the formater, I had issue to use eslint-ignore. Build issue again

Reply:
I understand that it could have been frustrating.
But it could be done like that 
<img width="798" alt="image" src="https://user-images.githubusercontent.com/41187899/163571642-5f931b70-5172-4388-ae78-b448b0e64368.png">
and that way, we don't suppress futur errors.